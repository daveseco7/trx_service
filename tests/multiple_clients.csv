type,client, tx,amount
deposit,1,1,1.0
deposit,2,10,5.0
deposit,1,2,1.0
deposit,3,30,5.0
deposit,2,11,1.0
withdrawal,2,12, 9.0
deposit,1,3,1.0
deposit,3,31,1.0
withdrawal,1,4, 2.0
withdrawal,2,13, 2.0
dispute,1,3
dispute,2,13
resolve,2,13
chargeback,2,10
resolve,1,3
deposit,1,5,1.0
withdrawal,3,32, 9.0
withdrawal,3,33, 2.0
withdrawal,1,6, 1.0