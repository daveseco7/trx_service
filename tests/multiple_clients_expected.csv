client,available,held,total,locked
1,1,0.0,1,false
2,4,-5,-1,true
3,4,0.0,4,false
